frontend/app/src/components/home/CurrentChatMessages.svelte

```diff
@@ -507,7 +509,9 @@
     $: {
         if (chat.chatId !== currentChatId) {
             currentChatId = chat.chatId;
-            initialised = false;
+
+            // If chat.latestEventIndex < 0, then no events need to be loaded so we set initialized to true
+            initialised = chat.latestEventIndex < 0;
```

Comment:
This is not what `initialised` is intended to mean. It's supposed to mean that we have loaded the first chunk of events for the chat and is used to control when we start listening for scroll events.

Reply:
For an empty direct chat its fine to start listening to scroll events immediately because you can't scroll until there are some events.

Reply:
Yes it _seems_ fine but I don't think it was added to short circuit user initiated scroll events. I still think it's seems confusing, risky and unnecessary to redefine "initialised" to mean something else.